src/axis.rs

```diff
@@ -0,0 +1,85 @@
+//! Axis enums representing CSS Grid axis
```

Comment:
Should we move the flexbox axes (i.e. FlexDirection) into this file too?

Reply:
We could do. I've been trying to keep the `flex` and `grid` stuff separate in case we want to split it into separate crates and/or feature flag flexbox as well as grid. Currently everything in `axis` is CSS-specific, but not specific to any particular layout algorithm (CSS Grid doesn't have any non-standard axes).